EIPS/eip-831.md

```diff
@@ -0,0 +1,42 @@
+## Preamble
+
+    EIP: 831
+    Title: URL Format for Ethereum
+    Author: ligi <ligi@ligi.de>
+    Type: Standard Track
+    Category: ERC
+    Status: Draft
+    Replaces: 67
+    Created: 2018-01-15
+
+## Simple Summary
+
+A standard way of creating Ethereum URLs for various use-cases.
+
+## Abstract
+
+URLs embedded in QR-codes, hyperlinks in web-pages, emails or chat messages provide for robust cross-application signaling between very loosely coupled applications. A standardized URL format allows for instant invocation of the user's preferred wallet application.
+
+## Motivation
+
+The need for this ERC emerged when refining ERC-681. We need a container ERC that does not carry the weight of the use-cases and just serves the container for them.
+
+## Specification
+
+### Syntax
+
+Ethereum URLs contain "ethereum" in their schema (protocol) part and are constructed as follows:
+
+    request                 = "ethereum" ":" [ prefix "-" ] payload
+    prefix                  = STRING
+    payload                 = STRING
```

Comment:
Nitpicking:

How to distinguish
```
payload = "with-hyphen"
request = "ethereum:with-hyphen"
```
against
```
prefix = "with"
payload = "hyphen"
request = "ethereum:with-hyphen"
```
?

Perhaps, if `prefix` is omitted, `payload` can contain no `"-"`s?

Reply:
Good point. We can use the property of 681 that if the prefix is omitted the payload must start with 0x (be an address) - will add this constraint - thanks

Reply:
Dam - doing this I noticed that it is not working as the address can now also be a ens-domain. And AFAIK the ens-domain can contain a "-" - so this is really a problem - think then we really need to prefix ens - think this is a good idea anyway for future updates - @nagydani - what do you think - or do you have a better idea?